switools/verifyswi.py

```diff
@@ -59,7 +65,7 @@ class VERIFY_SWI_RESULT:
    ERROR_INVALID_SIGNING_CERT = 9
    ERROR_INVALID_ROOT_CERT = 10
 
```

Comment:
```suggestion
   ERROR_MISSING_SWADAPT = 11
```

Reply:
latest push uses "ERROR_NO_SWADAPT_UTIL"
(your comments show in the middle of my commits diffs, this is so confusing... I thought I saw all your comments, then found some more of them while I looked at how my latest diffs looked on the web page).